src/Raven.Studio/wwwroot/App/views/manage/adminLogs.html

```diff
@@ -81,7 +81,8 @@ <h4>Log filters</h4>
                     <span class="caret"></span>
                 </button>
                 <div class="dropdown-menu colselect-dropdown slidein-style" data-bind="dropdownPanel: true">
-                    <div class="columns-list-container" data-bind="with: onDiskConfiguration" id="settingsContainer">
+                    <div class="dropdown-extended" data-bind="with: onDiskConfiguration" id="settingsContainer">
+                        <div class="bg-warning padding padding-xs text-center"><i class="icon-warning"></i> <span>Logs setting reset after server restart</span></div>
```

Comment:
* Logs settings will reset to default values after a server restart

or just:

* Logs settings will reset after a server restart